instance-configs/Config.cprobe.groovy

```diff
@@ -0,0 +1,716 @@
+/*
+ * NOTE
+ * ----
+ * This configuration assumes that the development environment will be used with
+ * run-app and the production environment will be used with the application
+ * packaged as a WAR and deployed to tomcat. Running grails run-war or otherwise
+ * running a WAR with the development profile set up or activating the
+ * production environment when running grails run-app are scenarios that have
+ * NOT been tested.
+ */
+quickStartURL = "../../files/NEPTUNE_tranSMART_Quick_Start_Guide.pdf"
```

Comment:
Should we point this at a CPROBE quickstart guide?

Reply:
Don't think there is one. Let's check with Becky.